en-US/T.xml

```diff
@@ -92,16 +92,24 @@
 			<term>terminal</term>
 			 <listitem>
 				<para>
-					<emphasis>n.</emphasis> Do not use to describe where to type commands. Use "command line" instead.
+					<emphasis>n.</emphasis> Use "terminal" to refer to a text-based interface where typed instructions are entered, or to refer to a session of this type. 
+					For example, "Open a terminal and log in to your OpenShift cluster", or "The exercise file captures any error messages from your terminal". 
 				</para>
-				 <para>
+				<para>
+					Use "command line" or "shell prompt" instead to refer to the prompt where commands are entered. 
```

Comment:
Can "command line" and "shell prompt" be used interchangeably? I suspect not, but the framing here could suggest otherwise. 

Initially, I was going to suggest adding a version of this definition (in line 99) to the "shell prompt" entry. But, looking at the current entry, I'm not sure whether they're entirely consistent. 

Current entry for shell prompt:
> Refers to the character at the beginning of the command line, and indicates that the shell is ready to accept commands. Do not use "command prompt", "terminal", or "shell".

To me, there's an apparent contradiction between the definition for "shell prompt" in the "terminal" entry, and the definition in the "shell prompt" entry. The "terminal" entry suggests that "command line" and "shell prompt" refer to the same thing (and thus could, by extension, perhaps be used interchangeably). On the other hand, the "shell prompt" entry says not to use "command prompt", suggesting that these two prompts are different. 

I do think we should follow the general principle that if we define a term in one entry, then we should include the same information in the "main" entry for the term in question. However, I understand that this quickly becomes complicated when dealing with a set of related terms. As a minimum viable product, though, the terms in question are: "terminal", "command line", and "shell prompt" (based on this update). Perhaps we can align these three entries in this update (and not worry about other related terms at the moment)?